### PR TITLE
Fix hound config so Jason doesn't rage as much

### PR DESCRIPTION
**Pull request by SamLau95**
See https://github.com/calblueprint/foodshift/pull/103 and https://github.com/calblueprint/foodshift/pull/94

### CODE REVIEW

**Comment by varunrau** [comment]
@calblueprint/worldreader wants some love

File: .rubocop.yml [hunk 1]
```diff
@@ -13,8 +15,21 @@ Metrics/LineLength:
   Description: 'Limit lines to 110 characters.'
   Max: 110
 
+Metrics/MethodLength:
+  Description: 'Limit methods to 15 lines.'
+  Max: 15
```
Comment (noahsark769) [inline .rubocop.yml]: 15 line limit on methods? Seems pretty short, doesn't it?

**Comment by noahsark769** [comment]
This is a matter of utmost importance for everyone watching this repo :)

File: .rubocop.yml (hunk 1 above)
Comment (SamLau95) [inline .rubocop.yml]: I've almost never had an issue with it when my methods are only doing one thing (which should be the case)

Comment (noahsark769) [inline .rubocop.yml]: Dang alright, fair enough

**Comment by akakitani** [comment]
Wait actually while you're at this Sam could you add an exclude for everything under `vendor/` too?? THANKS SAM YOU'RE THE BEST :smile_cat: :smile_cat: :smile_cat:

**Comment by SamLau95** [comment]
I gotchu @akakitani

**Pull request merged**